hsds/group_sn.py

```diff
@@ -177,13 +177,16 @@ async def POST_Group(request):
     aclCheck(app, domain_json, "create", username)
 
     verifyRoot(domain_json)
+    root_id = domain_json["root"]
 
-    link_id = None
-    link_title = None
+    # allow parent group creation or not
+    implicit = getBooleanParam(params, "implicit")
```

Comment:
Whether or not to create intermediate groups is specified in the creation properties of a group - see docs [here](https://docs.hdfgroup.org/hdf5/develop/group___l_c_p_l.html#ga66c4c5d3f34e5cf65d00e47a5387383c). Would it be possible to read this from provided creation properties instead?